torch/_tensor_docs.py

```diff
@@ -5196,6 +5196,10 @@ def callable(a, b) -> number
     has the correct :class:`torch.dtype` and :class:`torch.device`, then ``self`` is returned.
     Otherwise, the returned tensor is a copy of ``self`` with the desired
     :class:`torch.dtype` and :class:`torch.device`.
+    If ``self`` requires gradients (``requires_grad=True``) but the target
+    ``dtype`` specified is an integer type, the returned tensor will implicitly
+    set ``requires_grad=False``. This is because only Tensors of floating
+    point and complex dtype can require gradients.
```

Comment:
dtypes?

Reply:
Thanks for the review! Actually I copied it from here haha

https://github.com/pytorch/pytorch/blob/604467de208646f0c3b2663e45f2ff6a655a6716/torch/csrc/autograd/variable.h#L270

Reply:
Yea I agree it should be dtypes. You can change it there too 😛